net-p2p/bitcoin-qt/bitcoin-qt-0.20.1.ebuild

```diff
@@ -99,6 +99,8 @@ src_prepare() {
 		eapply "${knots_patchdir}/${KNOTS_P}.ts.patch"
 	fi
 
+	eapply "${FILESDIR}/${P}-boost-1.77-compat.patch"
```

Comment:
Looks like 0.20.1 is broken even with this patch, all the other versions seem to work. Can I just remove 0.20.1 after merging this PR to "solve" this?
```
/var/tmp/portage/vbslogs/build/net-p2p/bitcoin-qt-0.20.1:20211116-092610.log: * ERROR: net-p2p/bitcoin-qt-0.20.1::gentoo failed (compile phase):
/var/tmp/portage/vbslogs/build/net-p2p/bitcoind-0.20.1:20211116-093726.log: * ERROR: net-p2p/bitcoind-0.20.1::gentoo failed (compile phase):
/var/tmp/portage/vbslogs/build/net-p2p/bitcoin-qt-0.20.1:20211116-092501.log: * ERROR: net-p2p/bitcoin-qt-0.20.1::gentoo failed (compile phase):
```
```
        "atom": "=net-p2p/bitcoind-0.20.1",
        "emerge_cmdline": "emerge --verbose y --oneshot --usepkg-exclude net-p2p/bitcoind --deep --backtrack 300 =net-p2p/bitcoind-0.20.1",
        "emerge_default_opts": "--autounmask=y --autounmask-continue --autounmask-use=y --autounmask-write --buildpkg-exclude virtual --buildpkg-exclude acct-* --buildpkg-exclude sys-kernel --nospinner --jobs=8 --usepkg",
        "exit_code": 1,
        "features": "assume-digests binpkg-docompress binpkg-dostrip binpkg-logs binpkg-multi-instance buildpkg config-protect-if-modified distlocks ebuild-locks fixlafiles ipc-sandbox merge-sync multilib-strict network-sandbox news parallel-fetch parallel-install pid-sandbox preserve-libs protect-owned qa-unresolved-soname-deps sandbox sfperms strict strict-keepdir unknown-features-warn unmerge-logs unmerge-orphans userfetch userpriv usersandbox usersync xattr",
        "test_feature_toggle": false,
        "time": {
            "finished": "2021-11-16T09:38:33",
            "started": "2021-11-16T09:37:24"
        },
        "use_flags": "asm examples knots system-leveldb upnp wallet zeromq"
```
```
        "atom": "=net-p2p/bitcoin-qt-0.20.1",
        "emerge_cmdline": "emerge --verbose y --oneshot --usepkg-exclude net-p2p/bitcoin-qt --deep --backtrack 300 =net-p2p/bitcoin-qt-0.20.1",
        "emerge_default_opts": "--autounmask=y --autounmask-continue --autounmask-use=y --autounmask-write --buildpkg-exclude virtual --buildpkg-exclude acct-* --buildpkg-exclude sys-kernel --nospinner --jobs=8 --usepkg",
        "exit_code": 1,
        "features": "assume-digests binpkg-docompress binpkg-dostrip binpkg-logs binpkg-multi-instance buildpkg config-protect-if-modified distlocks ebuild-locks fixlafiles ipc-sandbox merge-sync multilib-strict network-sandbox news parallel-fetch parallel-install pid-sandbox preserve-libs protect-owned qa-unresolved-soname-deps sandbox sfperms strict strict-keepdir unknown-features-warn unmerge-logs unmerge-orphans userfetch userpriv usersandbox usersync xattr",
        "test_feature_toggle": false,
        "time": {
            "finished": "2021-11-16T09:27:17",
            "started": "2021-11-16T09:26:07"
        },
        "use_flags": "asm dbus kde knots qrcode -system-leveldb -upnp wallet zeromq"
```
```
        "atom": "=net-p2p/bitcoin-qt-0.20.1",
        "emerge_cmdline": "emerge --verbose y --oneshot --usepkg-exclude net-p2p/bitcoin-qt --deep --backtrack 300 =net-p2p/bitcoin-qt-0.20.1",
        "emerge_default_opts": "--autounmask=y --autounmask-continue --autounmask-use=y --autounmask-write --buildpkg-exclude virtual --buildpkg-exclude acct-* --buildpkg-exclude sys-kernel --nospinner --jobs=8 --usepkg",
        "exit_code": 1,
        "features": "assume-digests binpkg-docompress binpkg-dostrip binpkg-logs binpkg-multi-instance buildpkg config-protect-if-modified distlocks ebuild-locks fixlafiles ipc-sandbox merge-sync multilib-strict network-sandbox news parallel-fetch parallel-install pid-sandbox preserve-libs protect-owned qa-unresolved-soname-deps sandbox sfperms strict strict-keepdir unknown-features-warn unmerge-logs unmerge-orphans userfetch userpriv usersandbox usersync xattr",
        "test_feature_toggle": false,
        "time": {
            "finished": "2021-11-16T09:26:07",
            "started": "2021-11-16T09:24:59"
        },
        "use_flags": "asm dbus -kde knots -qrcode -system-leveldb -upnp wallet zeromq"
```
(it does seem to be related to `zeromq` USE flag)